### DISCUSSION TITLE
2021/04/03/clash/

### DISCUSSION THREAD

**Discussion by giscus** [xzsk2/xzsk2.github.io#2] [Announcements]
# 2021/04/03/clash/

症状在使用Chrome+SwitchyOmega+Clash的环境下，SwitchyOmega的“代理”操作会将代理请求发送至Clash，由Clash判断是否使用代理。这样就造成了用户想要某个域名走代理，但是Clash规则中默认该域名直连，最终还是直连的情况。

https://sakari.top/2021/04/03/clash/

**Comment by MaxNirvana** [comment]
请问如果浏览器是edge怎么办呢？

**Reply by pluto0x0** [reply]
进程名`chrome.exe`改为`msedge.exe`就行

**Reply by MaxNirvana** [reply]
多谢
